task_5/Toma.Lobanova/src/main/java/ua/od/hillel/BinaryNumber/BinaryNumber.java

```diff
@@ -0,0 +1,21 @@
+package ua.od.hillel.BinaryNumber;
+
+
+public class BinaryNumber {
+    public static void main(String...args){
+        int number=3456;
+        StringBuilder builder=new StringBuilder();
+        int pow = 0;
+        for (int i = 2; i <= number; i *= 2, pow ++) {
```

Comment:
I respect your sense of art, Toma ;)
but using the loop in that way is not a good way to do this operation